aptos-move/framework/Cargo.toml

```diff
@@ -31,15 +31,21 @@ bcs = { workspace = true }
 better_any = { workspace = true }
 blake2-rfc = { workspace = true }
 blst = { workspace = true }
+bulletproofs = { workspace = true }
+byteorder = { workspace = true }
 clap = { workspace = true }
 codespan-reporting = { workspace = true }
-curve25519-dalek = { workspace = true }
+# We need `zkcrypto/curve25519-dalek-ng` here instead of the normal `dalek-cryptography/curve25519-dalek` package. This
+# is because the `dalek-cryptography/bulletproofs` works with version 2 of `dalek-cryptography/curve25519-dalek` and
+# we work with version 3. Instead, we use `zkcrypto/bulleproofs` which relies on `zkcrypto/curve25519-dalek-ng`.
+curve25519-dalek = { package = "curve25519-dalek-ng", version = "4" }
```

Comment:
@JoshLind, IIRC, you did the Cargo.toml refactoring... is not using `workspace = true` here kosher? I need to override `curve25519-dalek` with `curve25519-dalek-ng` here.

(Note that the `curve25519-dalek-ng` package is included in the top-level `Cargo.toml`.)

Reply:
Yeah, this seems like a reasonable thing to do, i.e., the comment is clear to me that we can't reuse the `workspace` package because of the inconsistencies. So, looks okay to me 😄